paasta_tools/cli/cmds/validate.py

```diff
@@ -105,12 +110,16 @@ def validate_schema(file_path, file_type):
     except IOError:
         paasta_print('%s: %s' % (FAILED_READING_FILE, file_path))
         return False
-    if extension == '.yaml':
-        config_file_object = yaml.safe_load(config_file)
-    elif extension == '.json':
-        config_file_object = json.loads(config_file)
-    else:
-        config_file_object = config_file
+    try:
+        if extension == '.yaml':
+            config_file_object = yaml.safe_load(config_file)
+        elif extension == '.json':
+            config_file_object = json.loads(config_file)
+        else:
+            config_file_object = config_file
+    except: 
```

Comment:
Can you combine this and take advantage of the existing except in 110?

Just try to open and read the file on the same try